apps/condo/domains/acquiring/schema/RegisterMultiPaymentForInvoicesService.js

```diff
@@ -237,13 +241,21 @@ const RegisterMultiPaymentForInvoicesService = new GQLCustomSchema('RegisterMult
                 const amountToPay = Big(totalAmount.amountWithoutExplicitFee)
                     .add(Big(totalAmount.explicitServiceCharge))
                     .add(Big(totalAmount.explicitFee))
+
                 if (acquiringIntegration.minimumPaymentAmount && Big(amountToPay).lt(acquiringIntegration.minimumPaymentAmount)) {
                     throw new GQLError({
                         ...ERRORS.PAYMENT_AMOUNT_LESS_THAN_MINIMUM,
                         messageInterpolation: { minimumPaymentAmount: Big(acquiringIntegration.minimumPaymentAmount).toString() },
                     }, context)
                 }
 
+                if (acquiringIntegration.maximumPaymentAmount && Big(amountToPay).gt(acquiringIntegration.maximumPaymentAmount)) {
```

Comment:
Why not lte and gte
Minimum payment amount 15.00 - will throw an error that Payment amount must be at least 15.00